backend/app/routes/emailtest/control.js

```diff
@@ -1,11 +1,19 @@
-const service = require('./emailservice');
+const to = require('await-to-js').default;
+const sendEmail = require('../../../helpers/mailer');
 const router = require('express').Router({ mergeParams: true });
 router.post('/', email);
 
 async function email(req, res, next) {
-    await service.sendVerificationEmail(req.body.toEmail, req.body.fromEmail)
-        .then(() => res.json({message: `Email Sent!`}))
-        .catch(next);
+    [err] = await to (sendEmail({
+        from : req.body.fromEmail,
+        to: req.body.toEmail
+    })
+    );
+    if(err){
+        return next(err);
```

Comment:
error handling is not proper! Just check the response if you deliberately input wrong config credentials

Reply:
this is still pending

Reply:
Error handling is still not as per our codebase. You are expected to follow our coding guidelines.

Reply:
You still need to make this err refactored in our format